### PR TITLE
Bump bitcoin to pull package version 0.16.3 from bitcoin.org

### PR DESCRIPTION
**Pull request by mecampbellsoup**
https://bitcoin.org/bin/bitcoin-core-0.16.3/

### CODE REVIEW

**Comment by mecampbellsoup** [comment]
Hm, why is it showing the PR's diff as a conflict to be resolved I wonder?

**Comment by fxcoudert** [comment]
Because your branch is not based on master:

```
[fx:/usr … homebrew-core/Formula] master ± head bitcoin.rb 
class Bitcoin < Formula
  desc "Decentralized, peer to peer payment network"
  homepage "https://bitcoin.org/"
  url "https://bitcoin.org/bin/bitcoin-core-0.16.2/bitcoin-0.16.2.tar.gz"
  sha256 "1910c58b7d8e13acbd05b0e8fd9b4b61cbac5776f058ee10bc80c1d2a7c707d8"

  bottle do
    cellar :any
    sha256 "7bf881724620ca2fe46e336af30236fbeecdbcb7d158bc04d86ea34c7e7f83ff" => :high_sierra
    sha256 "9b6fe31e4ac46d30fb87bb72740ff9df0366547177ac57a845c9c998e24d340d" => :sierra
```

File: Formula/bitcoin.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Bitcoin < Formula
   desc "Decentralized, peer to peer payment network"
   homepage "https://bitcoin.org/"
-  url "https://bitcoin.org/bin/bitcoin-core-0.16.2/bitcoin-0.16.2.tar.gz"
-  sha256 "1910c58b7d8e13acbd05b0e8fd9b4b61cbac5776f058ee10bc80c1d2a7c707d8"
+  url "https://bitcoin.org/bin/bitcoin-core-0.16.3/bitcoin-0.16.3.tar.gz"
+  sha256 "836eed97dfc79cff09f356e8fbd6a6ef2de840fb9ff20ebffb51ccffdb100218"
```
Comment (mecampbellsoup) [inline Formula/bitcoin.rb]:
```
λ MEC-MBP homebrew-core → λ git bump-bitcoin-to-16-dot-3 → curl -sL https://bitcoin.org/bin/bitcoin-core-0.16.3/bitcoin-0.16.3.tar.gz | shasum -a256
836eed97dfc79cff09f356e8fbd6a6ef2de840fb9ff20ebffb51ccffdb100218  -
```

**Comment by mecampbellsoup** [comment]
Hey guys, did you see that Travis failed? https://travis-ci.com/mecampbellsoup/homebrew-core/builds/85376428?utm_medium=notification&utm_source=email

Looks like an env variable is undefined or something perhaps but not sure.

**Comment by fxcoudert** [comment]
Homebrew CI was all green: https://jenkins.brew.sh/job/Homebrew%20Core%20Pull%20Requests/30682/

**Pull request closed without merging**